recogs_examples_in_rasp.py

```diff
@@ -154,55 +180,63 @@ def get_percentages_with_ci_groupby_binary_data(df, groupby_key, alpha=0.05, pri
     ci_highs[idx] = ci[1]*100.0
     p[idx] = float(k)/float(n)*100
     if print_result:
-      print(f"{idx}: {p[idx]:0.2f}% ({(1-alpha)*100:0.2f}% confidence interval: {ci_lows[idx]:0.2f}% to {ci_highs[idx]:0.2f}% ({k} out of {n})")
+      print_stdout_and_file(f"{idx}: {p[idx]:0.2f}% ({(1-alpha)*100:0.2f}% confidence interval: {ci_lows[idx]:0.2f}% to {ci_highs[idx]:0.2f}% ({k} out of {n})", file_handle)
```

Comment:
Before this PR submits, will fix the pretty-printing by evaluation script (not the model itself) of the per category confidence intervals which are printed without a trailing parenthesis (should be `))` but is `)`). Not a blocker for generalization set evaluation though.

Reply:
E.g. you can see the effect when printing result of first 20 rows (first evaluation for this branch, see notebook link in other comment below, first time ever including complement phrases, only run one other time for the original model without complement phrase support excluding those examples):
```
Semantic exact match score on first 20 of ReCOGS_pos gen:
100.00% or 20.0 out of 20 (95% confidence interval: 83.16% to 100.00%)

Semantic Exact Match % by category:
do_dative_to_pp_dative: 100.00% (95.00% confidence interval: 15.81% to 100.00% (2.0 out of 2)
obj_omitted_transitive_to_transitive: 100.00% (95.00% confidence interval: 2.50% to 100.00% (1.0 out of 1)
obj_pp_to_subj_pp: 100.00% (95.00% confidence interval: 29.24% to 100.00% (3.0 out of 3)
obj_to_subj_common: 100.00% (95.00% confidence interval: 15.81% to 100.00% (2.0 out of 2)
obj_to_subj_proper: 100.00% (95.00% confidence interval: 15.81% to 100.00% (2.0 out of 2)
only_seen_as_transitive_subj_as_unacc_subj: 100.00% (95.00% confidence interval: 2.50% to 100.00% (1.0 out of 1)
only_seen_as_unacc_subj_as_obj_omitted_transitive_subj: 100.00% (95.00% confidence interval: 15.81% to 100.00% (2.0 out of 2)
passive_to_active: 100.00% (95.00% confidence interval: 2.50% to 100.00% (1.0 out of 1)
pp_dative_to_do_dative: 100.00% (95.00% confidence interval: 15.81% to 100.00% (2.0 out of 2)
prim_to_obj_proper: 100.00% (95.00% confidence interval: 2.50% to 100.00% (1.0 out of 1)
prim_to_subj_common: 100.00% (95.00% confidence interval: 2.50% to 100.00% (1.0 out of 1)
prim_to_subj_proper: 100.00% (95.00% confidence interval: 15.81% to 100.00% (2.0 out of 2)
```